stdlib/2and3/lib2to3/pygram.pyi

```diff
@@ -0,0 +1,107 @@
+# Stubs for lib2to3.pygram (Python 3.6)
+
+from typing import Any
+from lib2to3.pgen2.grammar import Grammar
+
+class Symbols:
+    and_expr: int
```

Comment:
You use them all the time when manipulating an AST. This was the reason I added stubs in the first place. Even though they're technically not guaranteed to exist, I added them here because the module's name is `pygram` and its documented purpose is to "export the Python grammar and symbols". So I assume the symbols to be consistent with the current (3.6) Grammar.txt.

But you're right that `pattern_symbols` should be different. I'll make `python_symbols` and `pattern_symbols` fake subclasses then.